content/en/docs/reference/kubectl/conventions.md

```diff
@@ -50,7 +50,7 @@ You can create the following resources using `kubectl run` with the `--generator
 Generators other than `run-pod/v1` are deprecated.
 {{< /note >}}
 
-If you do not specify a generator flag, other flags prompt you to use a specific generator. The following table lists the flags that force you to use specific generators, depending on the version of the cluster:
+If you do not explicitly specify a generator, specifying certain other flags force use of a specific generator. The following table lists the flags that force you to use specific generators:
```

Comment:
@sftim , line 53, nits: 4x specify or specific. Could use `set` or `assign`?
If you do not explicitly specify a generator, a generator will be implicitly set by other flags. The following table lists the flags and the generators that will be activated: